### PR TITLE
CLDR-13756 Add inflection for units

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13756
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by macchiati** [comment]
For reviewers: There is a detailed description of the work in https://docs.google.com/document/d/1JCqzblmuKfQlZWStSUsn3KhGN3I8Ra539vcTjY2Irns/edit#

**Comment by macchiati** [comment]
For reviewers: There is a detailed description of the work in https://docs.google.com/document/d/1JCqzblmuKfQlZWStSUsn3KhGN3I8Ra539vcTjY2Irns/edit#

File: tools/java/org/unicode/cldr/util/data/PathDescription.txt [hunk 1]
```diff
@@ -103,6 +103,8 @@
 
 ^//ldml/numbers/minimalPairs/ordinalMinimalPairs\[@ordinal="([^"]*)"] ; Minimal pairs for ordinals. For more information, please see http://cldr.org/translation/getting-started/plurals#TOC-Minimal-Pairs.
 ^//ldml/numbers/minimalPairs/pluralMinimalPairs\[@count="([^"]*)"] ; Minimal pairs for plurals (cardinals). For more information, please see http://cldr.org/translation/getting-started/plurals#TOC-Minimal-Pairs.
+^//ldml/numbers/minimalPairs/caseMinimalPairs\[@case="([^"]*)"] ; Minimal pairs for cases used in the language. For more information, please see http://cldr.org/translation/getting-started/plurals#TOC-Minimal-Pairs.
+^//ldml/numbers/minimalPairs/caseMinimalPairs\[@gender="([^"]*)"] ; Minimal pairs for genders. For more information, please see http://cldr.org/translation/getting-started/plurals#TOC-Minimal-Pairs.
```
Comment (pedberg-icu) [inline tools/java/org/unicode/cldr/util/data/PathDescription.txt:107]: ldml/numbers/minimalPairs/caseMinimalPairs\[@gender

from the dtd, this should be
ldml/numbers/minimalPairs/genderMinimalPairs\[@gender

Comment (macchiati) [reply tools/java/org/unicode/cldr/util/data/PathDescription.txt]: Thanks, just checked in fix.

**Pull request merged**